TunnelVisionLabs.ReferenceAssemblyAnnotator/TunnelVisionLabs.ReferenceAssemblyAnnotator.targets

```diff
@@ -4,7 +4,12 @@
   <UsingTask TaskName="TunnelVisionLabs.ReferenceAssemblyAnnotator.AnnotatorBuildTask" AssemblyFile="$(ReferenceAssemblyAnnotatorBuildTaskPath)TunnelVisionLabs.ReferenceAssemblyAnnotator.dll" />
 
   <PropertyGroup>
-    <GenerateNullableAttributes Condition="'$(GenerateNullableAttributes)' == ''">true</GenerateNullableAttributes>
+    <_FrameworkIncludesNullableAttributes Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(TargetFrameworkVersion.Substring(1))' &gt;= '3.0'">true</_FrameworkIncludesNullableAttributes>
```

Comment:
💡 Can leverage the version comparison intrinsic from microsoft/msbuild#4911

```suggestion
    <_FrameworkIncludesNullableAttributes Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '3.0'))">true</_FrameworkIncludesNullableAttributes>
```

Reply:
Problems:

> error MSB4186: Invalid static method invocation syntax: "[MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '2.1')". Method '[MSBuild]::VersionGreaterThanOrEquals' not found.

https://ci.appveyor.com/project/sharwell/referenceassemblyannotator/builds/29824141/job/lflagyg1jqwrui21#L121

Reply:
Reverting since MSBuild 16.5 is in prerelease and not set up on the build server.